docs/src/getting_started.md

```diff
@@ -99,27 +142,97 @@ fm = gevfitbayes(data, :SeaLevel)
 
 
-The approximate covariance matrix of the parameter estimates can be obtained with the function [`parametervar`](@ref):
+The approximate covariance matrix of the parameter estimates can be obtained with the  [`parametervar`](@ref) function:
 ```@repl portpirie
 parametervar(fm)
 ```
 
+Confidence intervals on the parameter estimates can be obtained with the [`cint`](@ref) function:
+```@repl portpirie
+cint(fm)
+```
 
-### Diagnostic plots
+#### Diagnostic plots
 
-Several diagnostic plots for assessing the accuracy of the GEV model fitted to the Port Pirie data are can be shown with the function [`diagnosticplots`](@ref).
+Several diagnostic plots for assessing the accuracy of the GEV model fitted to the Port Pirie data are can be shown with the [`diagnosticplots`](@ref)function:
 
 ```@example portpirie
-fm = gevfit(data, :SeaLevel)
+set_default_plot_size(21cm ,16cm)
+diagnosticplots(fm)
+```
+
+The diagnostic plots consist in the probability plot (upper left panel), quantile plot (upper right panel), return level plot (lower left panel) and the density plot (lower right panel). These plots can be displayed separately using respectively the functions [`probplot`](@ref), [`qqplot`](@ref), [`returnlevelplot`](@ref) and [`histplot`](@ref).
+
+
+#### Return level estimation
+
+*T*-year return level estimate can be obtained using the function [`returnlevel`](@ref) on a `fittedEVA` object. The first argument is the fitted model, the second is the return period in years and the last one is the confidence level for computing the confidence interval.
 
+For example, the 100-year return level for the Port Pirie block maxima model and the corresponding 95% confidence interval can be estimated with this commands:
+
+```@repl portpirie
+r = returnlevel(fm, 100, .95)
+```
+
+where the return value can be accessed with
+```@repl portpirie
+r.value
+```
+
+and where the corresponding confidence interval can be accessed with
+```@repl portpirie
+r.cint
+```
+
+!!! note "Type-stable function"
+
+    In this example of a stationary model, the function returns a unit dimension vector for the return level and a vector containing only one vector for the confidence interval. The reason is that the function always returns the same type in the stationary and non-stationary case. The function is therefore [type-stable](https://docs.julialang.org/en/v1/manual/performance-tips/index.html#Write-%22type-stable%22-functions-1) allowing better performance of code execution.  
+
+To get the scalar return level in the stationary case, the following command can be used:
+```@repl portpirie
+r.value[]
+```
+
+To get the scalar confidence interval in the stationary case, the following command can be used:
```

Comment:
Same ("credible").

Reply:
Yes, in Bayesian inference the term credible interval is preferred.